toqito/matrices/gell_mann.py

```diff
@@ -117,6 +117,6 @@ def gell_mann(ind: int, is_sparse: bool = False) -> np.ndarray | scipy.sparse.cs
         raise ValueError("Gell-Mann index values can only be values from 0 to 8 (inclusive).")
 
     if is_sparse:
-        gm_op = scipy.sparse.csr_matrix(gm_op)
+        gm_op = scipy.sparse.csr_matrix(gm_op)  # pylint: disable=redefined-variable-type
```

Comment:
Hmm. I think instead of disabling the linter here, we want to actually listen and use a different variable name (instead of redefining it).

Reply:
Good catch! I did the lazy thing when it was a quick fix.